docs/how-to-guides/develop-a-tool/use-file-path-as-tool-input.md

```diff
@@ -0,0 +1,73 @@
+# Use file path as tool input
+
+User sometimes need to reference their local files within a tool to implement some specific logic. To simplify this, we've introduced the FilePath input type. This input type enables users to either select an existing file or create a new one, then pass it to a tool, enabling the tool to access the file's content.
+
+In this guide, we will provide a detailed walkthrough on how to use FilePath as a tool input. We will also demostrate the user experience when utilizing this type of tool within a flow.
+
+## Prerequisites
+
+As FilePath is currently in its preview phase, it is necessary for you to install the test version of the PromptFlow package:
+
+    ```bash
+    pip install "promptflow==0.1.0b8.dev2" --extra-index-url https://azuremlsdktestpypi.azureedge.net/promptflow
+    ```
+
+## Use file path as tool input
+
+Here we use [an existing tool package](../../../examples/tools/tool-package-quickstart/my_tool_package) as an example. If you want to create your owner tool, please refer to [create and use tool package](https://github.com/microsoft/promptflow/blob/main/docs/how-to-guides/develop-a-tool/create-and-use-tool-package.md#create-custom-tool-package).
+
+1. Add FilePath input for your tool, refer to [this tool](../../../examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_file_path_input.py).
```

Comment:
1. Add a `FilePath` input for your tool, like in [this example](../../../examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_file_path_input.py):